data/config.yml

```diff
@@ -193,6 +193,12 @@ vulnerability:
     "CVEs - Filter: Publish date":
       selectors:
         - '{}[data-ouia-subnav="cves"] button:contains("Publish date")'
+    "Reports - Link: Download PDF":
+      selectors:
+        - '{}a:contains("Download PDF")'
```

Comment:
Hey '{}a' is not right here... that means the scope and the a are on the same element.
You want '{} a' I think.

You only want to combine them (no space) if they are supposed to be matching on the same.